tests/workbenches/notebook-controller/test_spawning.py

```diff
@@ -0,0 +1,201 @@
+            notebook = loadDefaultNotebook(unprivileged_client, self.NTB_NAMESPACE, self.NTB_NAME, notebookImage)
+            function_resource_manager.createResourceWithoutWait(unprivileged_client, notebook)
+
+        with allure.step("Wait for Notebook pod readiness"):
+            lblSelector: str = f"app={self.NTB_NAME}"
+            PodUtils.waitForPodsReady(admin_client, self.NTB_NAMESPACE, lblSelector, 1)
+
+
+#
+#     @BeforeAll
+#     void deployDataScienceCluster() {
+#         if (Environment.SKIP_DEPLOY_DSCI_DSC) {
+#             LOGGER.info("DSCI and DSC deploy is skipped");
+#             return;
+#         }
+#         // Create DSCI
+#         DSCInitialization dsci = DscUtils.getBasicDSCI();
+#
+#         // Create DSC
+#         DataScienceCluster dsc = new DataScienceClusterBuilder()
+#                 .withNewMetadata()
+#                 .withName(DS_PROJECT_NAME)
+#                 .endMetadata()
+#                 .withNewSpec()
+#                 .withComponents(
+#                     new ComponentsBuilder()
+#                         .withWorkbenches(
+#                             new WorkbenchesBuilder().withManagementState(Workbenches.ManagementState.Managed).build()
+#                         )
+#                         .withDashboard(
+#                             new DashboardBuilder().withManagementState(Dashboard.ManagementState.Managed).build()
+#                         )
+#                         .withKserve(
+#                             new KserveBuilder().withManagementState(Kserve.ManagementState.Managed).build()
+#                         )
+#                         .withKueue(
+#                             new KueueBuilder().withManagementState(Kueue.ManagementState.Managed).build()
+#                         )
+#                         .withCodeflare(
+#                             new CodeflareBuilder().withManagementState(Codeflare.ManagementState.Managed).build()
+#                         )
+#                         .withDatasciencepipelines(
+#                             new DatasciencepipelinesBuilder().withManagementState(Datasciencepipelines.ManagementState.Managed).build()
+#                         )
+#                         .withModelmeshserving(
+#                             new ModelmeshservingBuilder().withManagementState(Modelmeshserving.ManagementState.Managed).build()
+#                         )
+#                         .withRay(
+#                             new RayBuilder().withManagementState(Ray.ManagementState.Managed).build()
+#                         )
+#                         .build())
+#                 .endSpec()
+#                 .build();
+#         // Deploy DSCI,DSC
+#         KubeResourceManager.getInstance().createOrUpdateResourceWithWait(dsci);
+#         KubeResourceManager.getInstance().createResourceWithWait(dsc);
+#     }
+# }
+
+
+def loadDefaultNotebook(client: DynamicClient, namespace: str, name: str, image: str) -> Notebook:
```

Comment:
functions should go under `utils.py` in the relevant dir

Reply:
done